src/Codeception/Lib/Di.php

```diff
@@ -148,6 +148,9 @@ protected function prepareArgs(ReflectionMethod $method, array $defaults = []):
             } else {
```

Comment:
I think, that `$parameter->isVariadic()` should be checked when parameter has no typehint too, above.

Reply:
I expected you to make that change :)